2010-09-09  Gerald Pfeifer  <gerald@pfeifer.com>
	    Andrew Pinski  <pinskia@gmail.com>

	PR target/40959	
	* config.host (ia64*-*-freebsd*): Set extra_parts.  Set tmake_file.


Index: libgcc/config.host
===================================================================
--- libgcc/config.host	(revision 163977)
+++ libgcc/config.host	(working copy)
@@ -347,6 +347,8 @@
 	tmake_file="ia64/t-ia64"
 	;;
 ia64*-*-freebsd*)
+	extra_parts="crtbegin.o crtend.o crtbeginS.o crtendS.o crtfastmath.o"
+	tmake_file="ia64/t-ia64"
 	;;
 ia64*-*-linux*)
 	extra_parts="crtbegin.o crtend.o crtbeginS.o crtendS.o crtfastmath.o"
